dream_layer_backend/dream_layer.py

```
@@ -567,6 +567,360 @@ def get_controlnet_models_endpoint():
+        )
+        
+        # Get the actual output file size and dimensions
+        output_size = None
+        grid_size = "Unknown"
+        if os.path.exists(output_path):
+            output_size = os.path.getsize(output_path)
+            try:
+                from PIL import Image
+                with Image.open(output_path) as img:
+                    grid_size = f"{img.width}×{img.height}"
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)